src/org/exist/backup/Main.java

```diff
@@ -65,159 +62,120 @@
+            .build();
+
+
+    /* user/pass arguments */
+    private static final Argument<String> userArg = stringArgument("-u", "--user")
+            .description("set user.")
+            .required()
+            .build();
+    private static final Argument<String> passwordArg = stringArgument("-p", "--password")
+            .description("set the password for connecting to the database.")
+            .required()
+            .build();
+    private static final Argument<String> dbaPasswordArg = stringArgument("-P", "--dba-password")
+            .description("if the backup specifies a different password for the admin user, use this option to specify the new password. Otherwise you will get a permission denied")
+            .build();
+
+
+    /* backup arguments */
+    private static final Argument<String> backupCollectionArg = stringArgument("-b", "--backup")
+            .description("backup the specified collection.")
+            .required()
+            .build();
+    private static final Argument<File> backupOutputDirArg = fileArgument("-d", "--dir")
+            .description("specify the directory to use for backups.")
+            .build();
+
+
+    /* restore arguments */
+    private static final Argument<File> restoreArg = fileArgument("-r", "--restore")
+            .description("read the specified __contents__.xml file and restore the resources described there.")
+            .build();
+    private static final Argument<Boolean> rebuildExpathRepoArg = optionArgument("-R", "--rebuild")
+            .description("rebuild the EXpath app repository after restore.")
+            .defaultValue(false)
+            .build();
+
+    private static Properties loadProperties() {
         // read properties
         final Properties properties = new Properties();
-
         try {
-            final Path propFile = ConfigurationHelper.lookup("backup.properties");
-            InputStream pin;
-
-            if (Files.isReadable(propFile)) {
-                pin = Files.newInputStream(propFile);
-            } else {
-                pin = Main.class.getResourceAsStream("backup.properties");
-            }
+            InputStream pin = null;
+            try {
+                final Path propFile = ConfigurationHelper.lookup("backup.properties");
 
-            if (pin != null) {
+                if (Files.isReadable(propFile)) {
+                    pin = Files.newInputStream(propFile);
+                } else {
+                    pin = Main.class.getResourceAsStream("backup.properties");
+                }
 
-                try {
+                if (pin != null) {
                     properties.load(pin);
-                } finally {
+                }
+
+            } finally {
+                if (pin != null) {
                     pin.close();
                 }
             }
-
-        } catch (final IOException ioe) {
+        } catch (final IOException e) {
+            e.printStackTrace();
```

Comment:
No, I wanted the fact the `backup.properties` file is missing to be a non-fatal error.